src/client/proximaStreamClient.ts

```diff
@@ -130,10 +131,10 @@ export class ProximaStreamClient {
     return endpoint;
   }
 
-  private getStreamConsumerClient(endpoint: string) {
+  private getStreamConsumerClient(endpoint: StreamEndpoint) {
     return (
-      this.clients[endpoint] ??
-      (this.clients[endpoint] = new StreamDBConsumerClient(endpoint))
+      this.clients[endpoint.uri] ??
+      (this.clients[endpoint.uri] = new StreamDBConsumerHttpClient(endpoint.httpUri ?? endpoint.uri)) // todo: throw?
```

Comment:
@ivanbenko is it better to throw exception here or maybe try to generate that uri by adding '-rest' to the deepest subdomain (streams.buh.proxima.one -> streams-rest.proxima.one)?